src/backend/api/url/url.controller.js

```diff
@@ -14,7 +14,7 @@ class Controller {
 
     createOne = async (req, res) => {
         try {
-            const urlRecaptcha = `${URL_RECAPTCHA}?secret=${SECRECT_KEY}&response=${req.body['g-recaptcha-response']}&remoteip=${req.connection.remoteAddress}`;
+            const urlRecaptcha = `${URL_RECAPTCHA}?secret=${SECRECT_KEY}&response=${req.body['recaptcha']}&remoteip=${req.connection.remoteAddress}`;
```

Comment:
Use queryString library or write yourself an utils function to stringify the object
```
const obj = {
   secret: SECRECT_KEY,
   response: req.body.recaptcha,
   remoteip: req.connection.remoteAddress
}
const urlRecaptcha = URL_RECAPTCHA + '?' + qs.stringify(obj);
```
https://www.npmjs.com/package/qs

Reply:
@kimchilee11